doc/source/install.rst

```diff
@@ -24,63 +27,5 @@ repository with::
     pip install git+git://github.com/geopandas/geopandas.git
 
 
-Dependencies
-------------
```

Comment:
I would keep the dependencies here as well. Not only contributors need to know this